service/src/users-api.js

```diff
@@ -78,8 +83,9 @@ exports.register = (app) => {
   // use middleware.Authz() as middleware in the route dispatch path
   app.get("/api/users", checkJwt, middleware.Authz(), async (req, res) => {
     const users = await directory.getUsers(req);
-    if (users) {
-      res.status(200).json(users);
+    if (!!users) {
```

Comment:
In this sample, I think the helper (directory.getUsers) should return the POJSO object and the route handler should be able to just call res.status.json().